utils/exploratory_data_analysis.py

```diff
@@ -1,5 +1,6 @@
 from typing import Dict
 import pandas as pd
+import matplotlib.pyplot as plt
```

Comment:
I believe we are using plotly for all the visualisations since it is interractive